pages/app/[teacherprofile].js

```diff
@@ -0,0 +1,50 @@
+import { useState, useEffect } from 'react';
```

Comment:
Nit: useEffect is being imported here even though it hasn't been implemented yet.
I know you guys talked about it but I think I think adding the dynamic route [teacherprofile] would be better in its own directory like how you have student. Just a little clearer what's going on

Reply:
Ty for catching this!